CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 ## Changelog
 
+### 0.17.4
+
+**Commit Delta**: [Change from 0.17.3 release](https://github.com/plus3it/watchmaker/compare/0.17.3...0.17.4)
+
+**Released**: 2020.02.28
+
+**Summary**:
+
+*   ash-linux-formula
+    -   Fix regular expressions to avoid key doubling
```

Comment:
Why relying on commit messages makes for poor changelogs... All of the ash-linux updates can be summarized in a single line, something like: 

> Updates custom STIGbyID baseline to address several scan findings.